.github/workflows/main.yml

```diff
@@ -8,7 +8,7 @@ jobs:
   build:
     strategy:
       matrix:
-        rust-version: ["1.54", "1.55", "1.56", "1.57"]
+        rust-version: ["1.56", "1.57"]
```

Comment:
Quoting from the PR:

> Implementation details are rather straight up, however, for the HashMap merge with the `serde_json::Value`, in order to not do a single insert I used a feature that was implemented in Rust version `1.56` as it can be seen [here](https://doc.rust-lang.org/std/collections/struct.HashMap.html#impl-From%3C%5B(K%2C%20V)%3B%20N%5D%3E).